mail-filter/rspamd/rspamd-2.2.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils pax-utils systemd tmpfiles user
+
+if [[ ${PV} == *9999 ]] ; then
+	EGIT_REPO_URI="https://github.com/vstakhov/rspamd.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/vstakhov/rspamd/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+DESCRIPTION="Rapid spam filtering system"
+HOMEPAGE="https://github.com/vstakhov/rspamd"
+LICENSE="Apache-2.0"
+SLOT="0"
+IUSE="blas cpu_flags_x86_ssse3 jemalloc +jit libressl pcre2"
+
+RDEPEND="dev-db/sqlite:3
+	dev-libs/glib:2
+	dev-libs/icu:=
+	dev-libs/libev
+	dev-libs/libsodium
+	dev-util/ragel
+	net-libs/libnsl
+	sys-apps/file
+	blas? ( sci-libs/openblas )
+	cpu_flags_x86_ssse3? ( dev-libs/hyperscan )
+	jemalloc? ( dev-libs/jemalloc )
+	jit? ( dev-lang/luajit:2 )
+	!libressl? ( dev-libs/openssl:0=[-bindist] )
+	libressl? ( dev-libs/libressl:0= )
+	pcre2? ( dev-libs/libpcre2[jit=] )
+	!pcre2? ( dev-libs/libpcre[jit=] )"
+DEPEND="${RDEPEND}"
+
+pkg_setup() {
+	enewgroup rspamd
+	enewuser rspamd -1 -1 /var/lib/rspamd rspamd
```

Comment:
Please migrate to the new GLEP-81 way of handling users and groups. Lots of examples recently how to do that, but you need to mail to gentoo-dev requesting UID+GID for this package.

Rest looks good.

Reply:
Done, here is the gentoo-dev request https://archives.gentoo.org/gentoo-dev/message/b1006c85f91a66e848b5d649af3f863d. I basically copied `metadata.xml` from `mail-filter/rspamd` and therefore made @croessner and @djc maintainers of `acct-{user,group}/rspamd`.